src/components/VideoPlayer/VideoPlayerControls.js

```diff
@@ -0,0 +1,119 @@
+    videoPlayerRef: PropTypes.object.isRequired,
+};
+
+const defaultProps = {};
+
+function VideoPlayerControls({duration, position, url, videoPlayerRef}) {
+    const {togglePlay, isPlaying, currentlyPlayingURL, updateCurrentlyPlayingURL} = usePlaybackContext();
+    const {showPopover} = useVideoPopoverMenuContext();
+    const [durationFormatted, setDurationFormatted] = useState('0:00');
+    const [shouldShowTime, setShouldShowTime] = useState(false);
+
+    const isCurrentlySet = currentlyPlayingURL === url;
+    const isCurrentlyPlaying = isCurrentlySet && isPlaying;
+
+    const togglePlayCurrentVideo = useCallback(() => {
+        if (!isCurrentlySet) {
+            updateCurrentlyPlayingURL(url);
+        } else {
+            togglePlay();
+        }
+    }, [isCurrentlySet, togglePlay, updateCurrentlyPlayingURL, url]);
+
+    const onLayout = (e) => {
+        setShouldShowTime(e.nativeEvent.layout.width > 250);
+    };
+
+    const enterFullScreenMode = useCallback(() => {
+        updateCurrentlyPlayingURL(url);
+        videoPlayerRef.current.presentFullscreenPlayer();
+    }, [updateCurrentlyPlayingURL, url, videoPlayerRef]);
+
+    const seekPosition = useCallback(
+        (newPosition) => {
+            videoPlayerRef.current.setStatusAsync({positionMillis: newPosition});
+        },
+        [videoPlayerRef],
+    );
+
+    useEffect(() => {
+        setDurationFormatted(convertMillisecondsToTime(duration));
+    }, [duration]);
+
+    return (
+        <Animated.View
+            style={[styles.videoPlayerControlsContainer]}
+            onLayout={onLayout}
+        >
+            <View style={[styles.videoPlayerControlsButtonContainer]}>
+                <View style={[styles.videoPlayerControlsRow]}>
+                    <IconButton
+                        src={isCurrentlyPlaying ? Expensicons.Pause : Expensicons.Play}
+                        fill="white"
+                        accessibilityLabel="play/pause"
+                        onPress={togglePlayCurrentVideo}
+                        style={{marginRight: 8}}
+                    />
+                    {shouldShowTime && (
+                        <View style={[styles.videoPlayerControlsRow]}>
+                            <Text style={[styles.videoPlayerText, {width: 30}]}>{convertMillisecondsToTime(position)}</Text>
+                            <Text style={[styles.videoPlayerText]}>/</Text>
+                            <Text style={[styles.videoPlayerText, {width: 30}]}>{durationFormatted}</Text>
+                        </View>
+                    )}
+                </View>
+                <View style={[styles.videoPlayerControlsRow]}>
+                    <VolumeButton style={{marginRight: 12}} />
+                    <IconButton
+                        src={Expensicons.Fullscreen}
+                        accessibilityLabel="fullsreen"
+                        onPress={enterFullScreenMode}
+                        style={{marginRight: 12}}
```

Comment:
We can use `styles` for that
```suggestion
                        style={styles.mr3}
```